### PR TITLE
EnvironmentCredential and DefaultTokenCredentialTests

### PR DESCRIPTION
**Pull request by catalinaperalta**
Added tests for EnvironmentCredential and DefaultTokenCredential.
Fixed some formatting.

### CODE REVIEW

File: sdk/azidentity/default_token_credential_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package azidentity
```
Comment (jhendrixMSFT) [inline sdk/azidentity/default_token_credential_test.go]: missing license header

File: sdk/azidentity/environment_credential_test.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
+package azidentity
```
Comment (jhendrixMSFT) [inline sdk/azidentity/environment_credential_test.go]: missing license header

File: sdk/azidentity/default_token_credential_test.go (hunk 1 above)
Comment (catalinaperalta) [reply sdk/azidentity/default_token_credential_test.go]: Added

File: sdk/azidentity/environment_credential_test.go (hunk 2 above)
Comment (catalinaperalta) [reply sdk/azidentity/environment_credential_test.go]: Added

**Pull request merged**